029-IoTEdge/Student/Challenge-01.md

```diff
@@ -46,7 +45,7 @@ Check for the updated list of supported regions here: [Device Streams: Regional
 1. [Azure IoT Hub](https://docs.microsoft.com/en-us/azure/iot-hub/)
 1. [Azure IoT Edge](https://docs.microsoft.com/en-us/azure/iot-edge/about-iot-edge?view=iotedge-2018-06)
 1. [Azure Industrial IoT modules](https://azure.github.io/Industrial-IoT/)
-1. [IoT Edge BackGround Deck for WTH](./assets/IoTHub_Edge.pptx)
+1. [IoT Edge Background Deck for WTH](../Coach/Presentations/IoTHub_Edge.pptx)
```

Comment:
Red flag!  This is a link to content in the Coach guide and a link that opens the WTH repo from the student guide.  This is a no no as per the WTH Author guide.   

You can link to the PPTX if you use a raw link that opens the PPT file in the browser without opening the WTH repo.  I will make that change.